templates/check_nagios.cfg.j2

```diff
@@ -3,3 +3,9 @@
 command[{{ i.command }}]={{ i.path }} {{ i.arguments }}
 {% endfor %}
 {% endif %}
+
+{%- if git_nrpe_checks is defined %}
+{% for line in git_nrpe_checks %}
+{{ line }}
```

Comment:
Would have been nicer to retain uniform structure in the template, i.e. still hardcode `command` here and get the dynamic contents from the hash. I'm going to approve this and leave it for the author to decide if this should be made more beautiful : )

Reply:
Ah! We don't have the approval step in this repo. I'll merge this after comment from @oscarkraemer .

Reply:
I will make improvement to this.

Reply:
Okay, I changed this. I think the change is an improvement overall. My master plan is to someday make a breaking change where I remove `nagios_plugins` and `opsview_plugins` variables.